frontend/public/components/console-notifier.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import { Firehose } from './utils';
+import { referenceForModel } from '../module/k8s';
+import { ConsoleNotificationModel } from '../models/index';
+
+const ConsoleNotifier_: React.FC<ConsoleNotifierProps> = ({obj: {data}, position}) => {
+  if (_.isEmpty(data)) {
+    return null;
+  }
+
+  return <React.Fragment>
+    {_.map(data, notification => (notification.spec.position === position || notification.spec.position === 'topBottom')
+      ? <div key={notification.metadata.uid}
+        className="co-global-notification"
+        style={{
+          backgroundColor: notification.spec.backgroundColor,
```

Comment:
I assume the `backgroundColor` and `color` ought to be optional?   I don't necessarily trust everyone to pick good color combos.
- [ ] `backgroundColor` and `color` optional
- [ ] Provide defaults for these if they are not specified
- [ ] Consider instead following a convention such as https://getbootstrap.com/docs/4.3/components/alerts/#examples and using config for `success`, `danger`, `warning`, `info` instead of open-ended color options?

Reply:
Let's make the properties optional and provide reasonable defaults, but we have a requirement that the exact colors need to be customizable.

Reply:
`backgroundColor` and `color` are already optional.  If not specified, the default `co-global-notification` styles are used.  These are really optional overrides of those defaults.